bpf/deserialization_to_bpf_map/deserialization_to_bpf_map.c

```diff
@@ -590,7 +595,7 @@ int deserial_update_elem(void *key, void *value)
 
     ret = get_map_fd_info(id, &map_fd, &info);
     if (ret < 0) {
-        LOG_ERR("invlid MAP_ID: %d\n", id);
+        LOG_ERR("invlid MAP_ID: %d, errno:%d\n", id, errno);
```

Comment:
```suggestion
        LOG_ERR("invlid MAP_ID: %d, get map failed:%d\n", id, errno);
```
I'm not sure what `fd` means, representing the actual operation in the logs allows for a better understanding of the error